src/current/_includes/v23.1/cdc/bare-envelope-cdc-queries.md

```diff
@@ -0,0 +1 @@
+{% if page.name == "cdc-queries.md" %}CDC queries{% else %}[CDC queries](cdc-queries.html){% endif %} use [`envelope=bare`](create-changefeed.html#envelope) message format by default. The `bare` message envelope will place the output of the `SELECT` clause at the top level of the message instead of under an `"after"` key. When there is additional information that the changefeed is sending, such as [`updated`](create-changefeed.html#updated-option) or [`resolved`](create-changefeed.html#resolved-option) timestamps, the messages will include a `crdb` field containing this information.
```

Comment:
```suggestion
{% if page.name == "cdc-queries.md" %}CDC queries{% else %}[CDC queries](cdc-queries.html){% endif %} use [`envelope=bare`](create-changefeed.html#envelope) message format by default. The `bare` message envelope places the output of the `SELECT` clause at the top level of the message instead of under an `"after"` key. When there is additional information that the changefeed is sending, such as [`updated`](create-changefeed.html#updated-option) or [`resolved`](create-changefeed.html#resolved-option) timestamps, the messages will include a `crdb` field containing this information.
```

Reply:
Would it be more clear to say "instead of below"? The usage of under here confuses me

Reply:
I think when we're talking about fields in a JSON message, under makes sense